_validate/addonVersion_schema.json

```
@@ -266,6 +266,16 @@
 				"title": "Discussion comment URL",
 				"type": "string"
 			},
+			"submissionTime": {
+				"$id": "#/properties/submissionTime",
+				"default": 0,
+				"description": "Timestamp in milliseconds, corresponding to the submission time of the add-on",
+				"examples": [
+					1723523492363.093
```

Lets make this an integer, there's no need for more specificity than MS

```suggestion
					1723523492363
```